### PR TITLE
deflake remainder of jax

### PR DESCRIPTION
**Pull request by jakevdp**

### CODE REVIEW

File: jax/api.py [hunk 1]
```diff
@@ -23,13 +23,14 @@
 arrays.
 """
 
-
+# TODO(vanderplas): define exactly what attributes we want in jax.api.
```
Comment (hawkinsp) [inline jax/api.py:26]: Note that `api` isn't a public name; the entire thing should be hidden in an underscored part of `jax`. There is already a shim that hides most of the symbols in `api`.

File: setup.cfg [hunk 2]
```diff
@@ -9,13 +9,4 @@ max-complexity = 18
...
-  ./jax/interpreters/*.py
-  ./jax/lax/*.py
-  ./jax/lib/*.py
-  ./jax/nn/*.py
-  ./jax/ops/*.py
-  ./jax/numpy/*.py
-  ./jax/scipy/*.py
-  ./jax/third_party/*.py
-  ./jax/tools/*.py
+  ./jax/*.py
```
Comment (hawkinsp) [inline setup.cfg:12]: Does this include subdirectories?

**Review by hawkinsp — APPROVED** [review]

File: jax/api.py (hunk 1 above)
Comment (jakevdp) [reply jax/api.py]: got it, removed.

File: setup.cfg (hunk 2 above)
Comment (jakevdp) [reply setup.cfg:12]: Yes

**Pull request merged**